HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -203,6 +216,7 @@ def customiseHLTforCMSSW(process, menuType="GRun", fastSim=False):
         process = customiseFor11183(process)
         process = customiseFor11497(process)
         process = customiseFor12044(process)
+        process = customiseFor11920(process)
```

Comment:
This should be under a new if statement ```if cmsswVersion >= "CMSSW_8_0":``` as it seems this will not be put in 76X.